modules/cli-installing-cli-rpm.adoc

```diff
@@ -43,21 +43,12 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
 ----
 
 . Enable the repositories required by {product-title} {product-version}.
-
-** For Red Hat Enterprise Linux 8:
 +
 [source,terminal]
 ----
 # subscription-manager repos --enable="rhocp-4.10-for-rhel-8-x86_64-rpms"
 ----
 
-** For Red Hat Enterprise Linux 7:
-+
-[source,terminal]
-----
-# subscription-manager repos --enable="rhel-7-server-ose-4.10-rpms"
```

Comment:
@zhouying7780 Since you said you tested oc with RHEL 9, should we add the step to install oc via RPM with RHEL 9? Do you happen to know what the correct repo name would be for RHEL 9?